core/src/test/scala/integration/kafka/api/AdminClientTest.scala

```diff
@@ -178,7 +178,7 @@ class AdminClientTest extends IntegrationTestHarness with Logging {
     // OffsetOutOfRangeException if offset > high_watermark
     assertEquals(DeleteRecordsResult(-1L, Errors.OFFSET_OUT_OF_RANGE.exception()), client.deleteRecordsBefore(Map((tp, 20))).get()(tp))
     // TimeoutException if response is not available within user-specified timeout
-    assertEquals(DeleteRecordsResult(-1L, Errors.REQUEST_TIMED_OUT.exception()), client.deleteRecordsBefore(Map((tp, 5L))).get(0, TimeUnit.MILLISECONDS)(tp))
+    assertEquals(DeleteRecordsResult(-1L, Errors.REQUEST_TIMED_OUT.exception()), client.deleteRecordsBefore(Map((tp, 8L))).get(0, TimeUnit.MILLISECONDS)(tp))
```

Comment:
It seems this test may still fail if the response returned before the future.get() is called, although unlikely. Personally I prefer remove this test because it is still subject to race condition. If we want to test `CompositeFuture` timeout, we can add a separate test on that.